Mathlib/Analysis/NormedSpace/HahnBanach/SeparatingDual.lean

```diff
@@ -73,6 +75,26 @@ section Field
 variable {R V : Type*} [Field R] [AddCommGroup V] [TopologicalSpace R] [TopologicalSpace V]
   [TopologicalRing R] [TopologicalAddGroup V] [Module R V] [SeparatingDual R V]
 
+-- this could generalize to CommRing R if we were to add a section
+theorem _root_.separatingDual_iff_injective : SeparatingDual R V ↔
+    Function.Injective (ContinuousLinearMap.coeLM (R := R) R (M := V) (N₃ := R)).flip := by
+  simp_rw [separatingDual_def, Ne, injective_iff_map_eq_zero]
+  refine forall_congr' fun v ↦ ?_
```

Comment:
I haven't checked in lean, but I'm guessing the syntax would be `congrm ∀ v, ?_`, which I think is more readable.

Reply:
It works, thanks!